nautobot/extras/templates/extras/inc/approval_buttons_column.html

```diff
@@ -8,15 +8,15 @@
         <i class="mdi mdi-history"></i>
     </a>
 {% endif %}
-{% if "approve" in buttons and have_permission %}
-    <span title='{% if not record.is_active_stage %}This stage is not active{% elif request.user in record.users_that_already_approved %}You already approved this stage{% else %}Approve this stage{% endif %}'>
+{% if "approve" in buttons and have_permission and record.is_active_stage %}
+    <span title='{% if request.user in record.users_that_already_approved %}You already approved this stage{% else %}Approve this stage{% endif %}'>
         <a href="{% url approval_route  pk=record.pk %}?return_url={{ request.path }}{{ return_url_extra }}" class="btn btn-success btn-xs{% if not record.is_active_stage or request.user in record.users_that_already_approved %} disabled{% endif %}">
             <i class="mdi mdi-check"></i>
         </a>
     </span>
 {% endif %}
-{% if "deny" in buttons and have_permission %}
-    <span title='{% if not record.is_active_stage %}This stage is not active{% elif request.user in record.users_that_already_approved %}You already approved this stage{% else %}Deny this stage{% endif %}'>
+{% if "deny" in buttons and have_permission and record.is_active_stage %}
+    <span title='{%if request.user in record.users_that_already_approved %}You already denied this stage{% else %}Deny this stage{% endif %}'>
```

Comment:
Seems like something's not quite right here - do we have or need a `record.users_that_already_denied` property to use here?

Reply:
Yeah, you are right. Though I think now with the `record.is_active_stage` check. The deny button will not show up in the UI if the user has already denied the stage. So I will remove this.